vars/runCleanupResource.groovy

```diff
@@ -38,7 +41,7 @@ def call(Map params, String region){
             ./docker/env/hydra.sh --execute-on-runner \${RUNNER_IP} clean-resources --post-behavior --test-id \$SCT_TEST_ID
         else
             echo "SCT runner IP file is empty. Probably SCT Runner was not created."
-            exit 1
+            ./docker/env/hydra.sh clean-resources --post-behavior --test-id \$SCT_TEST_ID
```

Comment:
@bentsi , are we going to support non-runner by longevity subroutines (`runSendEmail.groovy`, `runCollectLogs.groovy`, `runCleanupResource.groovy`) ?
If that is true we need to address it on other rooutines
if it is not true, we need to revert this change

Reply:
Currently, some jobs are not executed on runner (Gce backend), but it failed in cleaning the resource.
In this PR, I switched all (almost all) gce backend jobs to use sct runner, so it won't be a problem.

Reply:
@dkropachev yes, some jobs don't need SCT Runner, like artifact tests

Reply:
@amoskong, @bentsi , We need to decide how do we check in the pipeline routine weather it is running on runner or not.
It used to be that we relay on a backend.
Now you are changing it, to relay on content of `sct_runner_ip`, which I'am totally okay with.
But we need to have this logic be the same across all routines, right now we are patching only one, which is not ok.

Also if we are going for `sct_runner_ip` content, you need to redo the logic to reflect that, ie. to have:

```
    echo "Starting to clean resources ..."
    RUNNER_IP=\$(cat sct_runner_ip||echo "")
    if [[ -n "\${RUNNER_IP}" ]] ; then
        ./docker/env/hydra.sh --execute-on-runner \${RUNNER_IP} clean-resources --post-behavior --test-id \$SCT_TEST_ID
    else
        ./docker/env/hydra.sh clean-resources --post-behavior --logdir "`pwd`"
    fi
    echo "Finished cleaning resources."

```

Instead of 

```
    echo "Starting to clean resources ..."
    if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" ]]; then
        RUNNER_IP=\$(cat sct_runner_ip||echo "")
        if [[ -n "\${RUNNER_IP}" ]] ; then
            ./docker/env/hydra.sh --execute-on-runner \${RUNNER_IP} clean-resources --post-behavior --test-id \$SCT_TEST_ID
        else
            echo "SCT runner IP file is empty. Probably SCT Runner was not created."
            exit 1
        fi
    else
        ./docker/env/hydra.sh clean-resources --post-behavior --logdir "`pwd`"
    fi
    echo "Finished cleaning resources."
```